lib/cli.rb

```diff
@@ -154,6 +154,16 @@ def self.handle_error(e)
     true
   end
 
+  def self.check_container_name!(image, name)
+    if image.include?("/") && name.include?("/")
```

Comment:
@thardeck Wouldn't it make sense to validate the name using `SystemDescription.validate_name` instead of special casing the `/` case?